src/lt/objs/plugins.cljs

```diff
@@ -494,7 +494,7 @@
    (search-input this)])
 
 (defui source-button [plugin]
```

Comment:
So I was actually going to bring this up, but didn't want to make the PR fix two fairly different things.

Whilst I was implementing, and opening, this PR, it became confusing when using `source` and `website` interchangeably. I think it would be a good idea to refactor the wording, so that it matches what the UI shows? I don't mind doing this either :)